plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -272,7 +286,9 @@ export class PersonState {
         }
 
         const oldPerson = await this.db.fetchPerson(teamId, previousDistinctId)
-        const newPerson = await this.db.fetchPerson(teamId, distinctId)
+        // :TRICKY: Reduce needless lookups for person
```

Comment:
I'm aware of the context but this comment doesn't help much if the reader doesn't have this context. I mean as in why this is `TRICKY`

Reply:
Because we're basically modifying 'global' state (for this class) instead of relying on return values. It's bad code.

However the whole `handleIdentify` logic is so convoluted that I didn't want to get into extensive refactoring and risk breaking it. Instead I'm relying on extensive code coverage.

Reply:
Was a comment on the comment rather than what it's doing, but either way, no big deal